react/src/pages/utils/upload/DataEntryTable.tsx

```diff
@@ -313,6 +360,25 @@ function DataEntryCell(
     );
 }
 
+/**
+ * A data entry cell for columns which require a boolean value.
+ */
+function CheckboxCell(props: {
+    value: boolean;
+    onEdit: (newValue: boolean) => void;
+    disabled?: boolean;
+}) {
+    function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
+        props.onEdit(event.target.checked);
+    }
+
+    return (
+        <TableCell padding="checkbox">
+            <Checkbox checked={!!props.value} onChange={handleChange} />
```

Comment:
```suggestion
            <Checkbox checked={props.value} onChange={event => props.onEdit(event.target.checked)} />
```
This should shorten this component by a lot. `props.disabled` isn't used at all though.